ui-react/packages/atlasmap-ui/src/common/useSingleInputDialog.tsx

```diff
@@ -74,16 +74,23 @@ export function useSingleInputDialog({
           aria-label={title}
           isRequired={true}
           isValid={isValid}
+          data-testid={'input-text-field'}
         />,
         <Button
           key={'confirm'}
           variant={'primary'}
           onClick={handleConfirm}
           isDisabled={!isValid}
+          data-testid={'confirm-button'}
         >
           Confirm
         </Button>,
-        <Button key={'cancel'} variant={'link'} onClick={handleCancel}>
+        <Button
+          key={'cancel'}
+          variant={'link'}
+          onClick={handleCancel}
+          data-testid={'cancel-button'}
```

Comment:
Same with the above.

Reply:
sure, makes sense. Updated all the 'cancel-button' and 'confirm-button' test-id's